core/common/src/main/java/alluxio/Constants.java

```diff
@@ -111,9 +110,8 @@
   public static final String FILE_SYSTEM_WORKER_CLIENT_SERVICE_NAME = "FileSystemWorkerClient";
   public static final String KEY_VALUE_MASTER_CLIENT_SERVICE_NAME = "KeyValueMasterClient";
   public static final String KEY_VALUE_WORKER_CLIENT_SERVICE_NAME = "KeyValueWorkerClient";
-  public static final String FILE_SYSTEM_CLIENT_SERVICE_NAME = "FileSystemClient";
 
-  public static final String REST_API_PREFIX = "/v1/api";
+  public static final String REST_API_PREFIX = "/api/v1";
```

Comment:
as per suggestion by @aaudiber (I am fine with either and not reversing means note braking backwards compatibility with 1.3 REST API clients)